modules/microshift-ingress-controller-config.adoc

```diff
@@ -62,6 +77,28 @@ ingress:
 |===
 |Parameter |Description
 
+|`certificateSecret`
+|The `certificateSecret` value is a reference to a secret that contains the default certificate that is served by the Ingress Controller. When Routes do not specify their own certificate, `certificateSecret` is used.
+
+The secret must contain the following keys and data:
+* `tls.crt`: certificate file contents
```

Comment:
```suggestion

* `tls.crt`: certificate file contents
```
need a blank line here to properly display bullet points, currently:
<img width="570" alt="image" src="https://github.com/user-attachments/assets/2da08155-f64d-4429-804a-0620aa00a598" />